betterrolls-swade2/lang/en.json

```diff
@@ -501,6 +501,8 @@
   "BRSW.PopoutChatButton": "Add a button to the cards to popout",
   "BRSW.PopoutChatButtonHint": "Add another button to the head of the cards allowing you to popout easier",
   "BRSW.PopoutCard": "Popout this card",
+  "BRSW.No_Action_Message": "No actions selected message",
+  "BRSW.No_Action_Message_Hint": "Set the message to show the player's when no action was selected",
```

Comment:
Remove the underscore in Message_Hint to match existing naming convention.

Reply:
removed